app/views/static/home.html.erb

```diff
@@ -1,11 +1,187 @@
+          </div>
+        </div>
+        <div class="visible-xs col-xs-12 signup-form">
+          <div class="form-group">
+            <button type="submit" class="btn btn-default btn-block">Signup for Free</button>
+          </div>
+          <div calss="form-group">
+            <div class="hr-label">
+              <span class="hr-label__text">or</span>
+            </div>
+          </div>
+          <div class="form-group">
+            <button type="submit" class="btn btn-default btn-block">Signup for Free with Facebook</button>
+          </div>
+        </div>
+        <div class="hidden-xs col-xs-12 col-sm-5">
+          <h3 class="signup-text">Sign up</h3>
+          <form class="form-horizontal signup-form cover-signup-form">
+            <div class="form-group">
+              <div class="col-xs-12">
+                <input class="form-control" placeholder="Full Name">
+              </div>
+            </div>
+            <div class="form-group">
+              <div class="col-xs-12 col-md-6">
+                <input type="email" class="form-control" placeholder="Email">
+              </div>
+              <div class="col-xs-12 col-md-6">
+                <input type="password" class="form-control" placeholder="Password">
+              </div>
+            </div>
+            <div class="form-group">
+              <div class="col-xs-12">
+                <div class="checkbox">
+                  <label>
+                    <input type="checkbox"> I agree to the <a class="privacy-link"href="#">Erudition Terms</a>
+                  </label>
+                </div>
+              </div>
+            </div>
+            <div class="form-group">
+              <div class="col-xs-12">
+                <button type="submit" class="btn btn-default btn-block">Signup for Free</button>
+              </div>
+            </div>
+            <div calss="form-group">
+              <div class="col-xs-12 hr-label">
+                <span class="hr-label__text">or</span>
+              </div>
+            </div>
+            <div class="form-group">
+              <div class="col-xs-12">
+                <button type="submit" class="btn btn-default btn-block">Signup for Free with Facebook</button>
+              </div>
+            </div>
+          </form>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <section class="home-section home-section--gray">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12 col-sm-5">
+          <div class="home-section__thumb">
+            <%= image_tag('static/home-screenshot-01.jpg') %>
+          </div>
+        </div>
+        <div class="col-xs-12 col-sm-7">
+          <h2 class="home-section__text">
+            <small>Section One</small>
+            Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat
+          </h2>
+        </div>
+      </div>
+    </div>
+  </section>
```

Comment:
A matter of preference, but you might consider breaking this file into several partials just for maintainability. 150+ lines of HTML can sometimes feel unwieldy :)